api/server.py

```diff
@@ -51,6 +51,9 @@ def stampify_url():
     max_pages = request.args.get('max_pages')
     enable_animations = request.args.get('animations')
 
+    if not max_pages:
+        max_pages = 8
```

Comment:
Add comment here explaining that this is default value.

Reply:
Updated! Thanks for suggestion.